.github/workflows/publish-pr-check-content.yml

```diff
@@ -20,16 +20,16 @@ jobs:
     # only if it's a pull request
     if: github.event.workflow_run.event == 'pull_request'
     name: publish
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
       - name: download dist artifact
-        uses: dawidd6/action-download-artifact@v2
+        uses: dawidd6/action-download-artifact@v2.18.0
```

Comment:
Why not use https://github.com/dawidd6/action-download-artifact/releases/tag/v2.24.2 if you're going to hardcode a specific v2 version?

Also I don't know the answer to this, but will GH pull the latest 2.y.z if you set @v2, or do you have to set the specific version of 2.y.z to get that?

Reply:
Looking at https://github.com/actions/upload-artifact as an example it looks like
https://github.com/actions/upload-artifact/tree/v3.1.1 matches https://github.com/actions/upload-artifact/tree/v3

so yes v3  gets the latest v3.x.y. Should I hardcode it to the latest 3 digit version for traceability purposes?

Also re: why I used 2.18.0 I think my original intent was to use the first version that switched to node 16 so there would be the smallest amount of change/impact/potential breakage.

Reply:
I'm torn between the traceability of `vx.y.z` versions and the automatic updating of `vx`. 

Given we use `vx` versions today and have rarely had unexpected build failures... I think it's safe to keep that up. But if some day we decide to build a tool to fetch latest versions and update GH actions w/ `vx.y.z` and keep it current, we could switch then to the more declarative system.